apps/protocol-frontend/src/components/ContributionDetailShell.tsx

```diff
@@ -76,7 +76,7 @@ const ContributionDetailShell = ({
                   : `${contribution?.status.name} 👀`}
               </Badge>
               <Text fontSize="sm" color="gray.600">
-                {contribution?.date_of_engagement}
+                {`${contribution?.date_of_engagement}`}
```

Comment:
Why do we need this change?

Reply:
It's throwing the following error: 
`TS2322: Type 'string | Date' is not assignable to type 'ReactNode'`